gateway-js/src/__tests__/executeQueryPlan.test.ts

```diff
@@ -126,7 +126,9 @@ describe('executeQueryPlan', () => {
       overrideResolversInService('accounts', {
         RootQuery: {
           me() {
-            throw new AuthenticationError('Something went wrong');
+            throw new GraphQLError('Something went wrong', {
```

Comment:
Is losing the `name` here anything to worry about?

Reply:
I hope not? Note that it doesn't affect the JSON output (I think!)

Reply:
(It is also a test.)